nilearn/plotting/tests/test_find_cuts.py

```diff
@@ -72,6 +73,28 @@ def test_find_cut_slices():
                                n_cuts=n_cuts, spacing=2)
 
 
+def test_validity_of_ncuts_error_in_find_cut_slices():
+    data = np.zeros((50, 50, 50))
+    affine = np.eye(4)
+    x_map, y_map, z_map = 25, 5, 20
+    data[x_map - 15:x_map + 15, y_map - 3:y_map + 3, z_map - 10:z_map + 10] = 1
+    img = nibabel.Nifti1Image(data, affine)
+    for direction, n_cuts in zip('xyz', (0, -2, -10.00034)):
```

Comment:
You need a test with a positive number that raises an exception. That would have allowed you to catch the bug above.